presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/ThriftHiveMetastoreConfig.java

```diff
@@ -111,4 +112,17 @@ public ThriftHiveMetastoreConfig setMaxBackoffDelay(Duration maxBackoffDelay)
         this.maxBackoffDelay = maxBackoffDelay;
         return this;
     }
+
+    public boolean isImpersonationEnabled()
+    {
+        return impersonationEnabled;
+    }
+
+    @Config("hive.metastore.thrift.impersonation.enabled")
+    @ConfigDescription("Should Presto user be impersonated when communicating with Hive metastore")
```

Comment:
Maybe say "end user" rather than "Presto user" (since "Presto user" might be confused with the OS user of the Presto server process)

Also remove "Hive" since it's redundant and to be consistent with the other descriptions